pkg/osquery/tables/tablehelpers/exec.go

```diff
@@ -21,6 +26,11 @@ import (
 //
 // This is not suitable for high performance work -- it allocates new buffers each time.
 func Exec(ctx context.Context, logger log.Logger, timeoutSeconds int, possibleBins []string, args []string, includeStderr bool) ([]byte, error) {
+	ctx, span := traces.New(ctx, trace.WithAttributes(
+		attribute.StringSlice(traces.AttributeName("tablehelpers", "possible_binaries"), possibleBins),
+		attribute.StringSlice(traces.AttributeName("tablehelpers", "args"), args)))
+	defer span.End()
```

Comment:
This `traces.New(ctx, trace.WithAttributes(...)` feels awkward. 

We have this `traces` wrapper, which seems good. But then we also need to bring in otel for a bunch more, so that feels weird. I kinda think we should find ways to push more through our wrapper. 

I think we should consider the common usage, and then make that easy with the wrapper. Thinking aloud...
* do spans have names? if so `trace.New(ctx, name, ...)`.
* Are most of our attributes going to be `traces.AttributeName`? If so, we can take `keyvals` and process them out. 
* We probably sometimes need an escape value, so maybe a `New(ctx context.Context, keyvals ...interface{})` and then `NewRawOtel(ctx context.Context, opts ...trace.SpanStartOption)`

Reply:
I think more of a wrapper makes sense! I was waffling and not sure if it was optimizing too early.

Re: span name, I want to set the span name inside `traces.New`, rather than taking it as an arg, so that we can make sure we're setting span names consistently/avoid accidentally copy-pasting a different span's name.

Right now the only span options I'm using are `WithAttributes`, yeah -- it's possible we may want some of the others in the future.

Reply:
pushing name to the wrapper makes sense. 

Not sure which way is over/under optimizing. I'm looking at the diff, and seeing a lot of `otel` scattered about, and it just feels wrong.

Reply:
I've updated -- let me know how this looks to you.